agent/agentssh/agentssh.go

```diff
@@ -172,13 +172,15 @@ func (s *Server) sessionHandler(session ssh.Session) {
 
 	ctx := session.Context()
 
+	extraEnv := make([]string, 0)
 	x11, hasX11 := session.X11()
 	if hasX11 {
 		handled := s.x11Handler(session.Context(), x11)
 		if !handled {
 			_ = session.Exit(1)
 			return
 		}
+		extraEnv = append(extraEnv, fmt.Sprintf("DISPLAY=:%d", x11.ScreenNumber))
```

Comment:
```suggestion
		extraEnv = append(extraEnv, fmt.Sprintf("DISPLAY=:%d.0", x11.ScreenNumber))
```
I see this pattern mostly used these days, not sure if it'll make a difference or not but shouldn't hurt?